app/test/frontend/golden/pkg_versions_page.html

```diff
@@ -74,7 +74,7 @@ <h1 class="_visuallyhidden">Dart pub</h1>
 
 <main>
   
-<h1>All versions of foobar</h1>
+<h1>Stable versions of foobar</h1>
```

Comment:
Could you post a screen shot? Or update the template sample to render both flavors?

Reply:
deploying it to staging, check the following page in ~10 minutes: https://dartlang-pub-dev.appspot.com/packages/test/versions